### PR TITLE
style-refactor

### PR DESCRIPTION
**Pull request by kghamilton89**
Python Client Docs review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ connection = connect('/path/to/connection.properties')
 
 ## Logging
 
-Logging to `stdout` is **enabled** by default. To disable logging, modify the logger at the beginning of the script.
+Logging to `stdout` is **enabled** by default. To disable logging, append the `logger.disabled` setting the `logging` method.
```
Comment (rodionos) [inline README.md]: reads wrong, type?

File: README.md [hunk 2]
```diff
@@ -289,11 +289,11 @@ svc.insert(message)
...
 
-When querying series from the database, you need to pass the following filters to the `SeriesService`:
+When querying series in the database, pass the following filters to the `SeriesService`:
 
-- [`SeriesFilter`](./atsd_client/models/_data_queries.py#L267) requires specifying the metric name. You can also include data type (history or forecast), series tags, and other parameters.
-- [`EntityFilter`](./atsd_client/models/_data_queries.py#L126) can be set by providing entity name, names of multiple entities, or the name of the entity group or entity expression.
-- [`DateFilter`](./atsd_client/models/_data_queries.py#L162) can be set by specifying the `startDate`, `endDate`, or `interval` fields. Some **combination** of these parameters is required to establish a specific time range. The `startDate` and `endDate` fields can be provided either as keywords from [calendar syntax](https://axibase.com/docs/atsd/shared/calendar.html), an ISO 8601 formatted string, Unix milliseconds, or a Python datetime object.
+* [`SeriesFilter`](./atsd_client/models/_data_queries.py#L267): Requires specifying the metric name. Alternatively, include data type, series tags, and other parameters.
```
Comment (rodionos) [inline README.md]: Here and after, standardize to use present tense: filters, selects, retrieves, includes, specifies, contains, etc.

File: README.md (hunk 1 above)
Comment (kghamilton89) [reply README.md]: replaced.

File: README.md (hunk 2 above)
Comment (kghamilton89) [reply README.md]: All instances of gerund (continuous tense) verbs removed if possible.

**Comment by kghamilton89** [comment]
The explanations are much more specific now, your suggestions were very helpful.

**Pull request merged**